emails/templates/emails/wrapped_email.html

```diff
@@ -38,9 +42,16 @@
             <tr>
               <td width="100%" align="center" style="line-height: 150%;">
                 <p style="color: #363959; font-size: 13px; font-family: sans-serif; margin-top: 0; margin-bottom: 0;">
-                  This message was forwarded from <span style="font-family: sans-serif; font-weight: bolder; color: #20123a; text-decoration: none; font-size: 13px;">{{ display_email|safe }}</span> by <a href="{{ SITE_ORIGIN }}" target="_blank" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">Firefox Relay</a>.
+                  {% bold_violet_text display_email as bolded_email %}
+                  {% bold_violet_link SITE_ORIGIN 'Firefox Private Relay' as linked_origin %}
+                  {% ftlmsg 'forwarded-email-header-from' display_email=bolded_email linked_origin=linked_origin as email_wrap_forwarded_from %}
+                  {{ email_wrap_forwarded_from|safe }}
                   {% if has_attachment %}
-                    <br>Firefox Relay supports email forwarding (including attachments) of email up to 150KB in size. To learn more check our <a href="{{ faq_page }}" target="_blank" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">FAQ page</a>.
+                      {% ftlmsg 'nav-faq' as faq_text %}
+                      {% url 'faq' as faq_url %}
+                      {% bold_violet_link faq_url faq_text as faq_link %}
+                      {% ftlmsg 'forwarded-email-header-attachment' faq_link=faq_link as forwarded_email_wrap_attachment %}
```

Comment:
I added this [same comment](https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/13#discussion_r714404856) on the L10N PR, but I recommend passing the attachment size as a variable in this string. 

```suggestion
                      {% ftlmsg 'forwarded-email-header-attachment' faq_link=faq_link as forwarded_email_wrap_attachment size='150' unit='KB' %}
```